src/host/outputStream.cpp

```diff
@@ -286,7 +286,17 @@ void ConhostInternalGetSet::ShowWindow(bool showOrHide)
 {
     const auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
     const auto hwnd = gci.IsInVtIoMode() ? ServiceLocator::LocatePseudoWindow() : ServiceLocator::LocateConsoleWindow()->GetWindowHandle();
+
+    // GH#13301 - When we send this ShowWindow message, if we send it to it's
+    // going to need to get processed by the window message thread before

```

Comment:
```
    // GH#13301 - When we send this ShowWindow message, if we send it to it's
    // going to need to get processed by the window message thread before
```

send it to what? I can't parse this comment haha.

Reply:
guh I don't even know. Lemme rewrite that